dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -584,7 +581,15 @@ else if (_gotWriter == true) {
 // code above LIBERTY only
             }
             catch (Throwable th) {
-                logger.logp(Level.SEVERE, CLASS_NAME,"closeResponseOutput", "Error.while.closing.response.output",th);
+                //Client can close the connection and cause this closing error. Since there is no re-throw backup and the response is winding down,
+                //there is no beneficial other than this random alarm message(s) in console log.
+                //Since 6.1, log this under trace.
+                if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.SEVERE)) {
+                    logger.logp(Level.SEVERE, CLASS_NAME,"closeResponseOutput", "Error.while.closing.response.output",th);
```

Comment:
Is this one that we've had cases on etc?

Reply:
github issue. The workaround is to suppress it with the LOG property.